### ISSUE TITLE
Crash on missing config

### ISSUE THREAD

**Issue by exadeci** [onewheelskyward/lita-onewheel-forecast-io#11]
Thanks for working on this plugin.
When you comment 
**uri** or **api_key** in lita config your plugin crashes :

```
ERROR: Lita::Handlers::OnewheelForecastIo crashed. The exception was:
undefined method `[]' for nil:NilClass
```

And **lita_config_sample.rb** still has the old configuration maybe you could have a default value for the uri, being a forcast io plugin we expect it to conect to forcast.io but keeping the option is useful.
That file could be removed and/or you could add the two lines to the Readme.

**Comment by reidblomquist** [comment]
@exadeci thanks for the input! https://github.com/onewheelskyward/lita-onewheel-forecast-io/pull/13 should address the majority of your concerns voiced here. Not sure the crash behavior we're seeing here is a big deal as it's due to a mis/unconfigured, but will look into adding more graceful handling.

**Comment by exadeci** [comment]
It's just that there is a check here: https://github.com/reidblomquist/lita-forecast-io/blob/305c7c2137078d442ab562abe4e811339a222c14/lib/lita/handlers/utils.rb#L140
But I think it's unattainable because of the crash on start so I thought it could be moved earlier .

**Comment by onewheelskyward** [comment]
Yeah, that was a quick fix I threw in to keep me from bone-headedly spinning it up on new bots.  I can take another look at it and see if I can fix it for good.  Thanks for the issue report!
